drivers/volume/docker/volume.go

```diff
@@ -804,6 +832,22 @@ func (d *driver) volumeRootPath() string {
 	return d.r.Config.GetString("linux.volume.rootPath")
 }
 
+func (d *driver) volumeType() string {
+	return d.r.Config.GetString("docker.volumeType")
+}
+
+func (d *driver) iops() string {
+	return strconv.Itoa(d.r.Config.GetInt("docker.iops"))
+}
+
+func (d *driver) size() string {
+	return strconv.Itoa(d.r.Config.GetInt("docker.size"))
```

Comment:
Hi @clintonskitson,

Please use the `GetString` function directly instead of getting it as an int and casting it to a string. The underlying code should handle the casting for you. I think.